### ISSUE TITLE
Rename RHEL3u2 rpmlists' name to standard naming convention

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#177]
**Reported by bli on 13 Jun 2006 02:21 UTC**
Currently in [oscarsamples](browser:trunk/oscarsamples), there are rpmlists for RHEL3u2 - we need to determine whether they are necessary anymore and if not, delete them.

**Comment by dikim33** [comment]
**Comment by dikim on 3 Aug 2006 00:55 UTC**
I confirm that redhat-3asU2-i386.rpmlist is not used on my rhel3-x86 (u6) but rhel-3-i386.rpmlist is used instead.

So, I think we can safely remove the list redhat-3asU2-i386.rpmlist for now.

There is another list "redhat-3asU2-ia64.rpmlist" to confirm but I believe that rhel-3-ia64.rpmlist will be used instead too.

If anyone with ia64 machine can confirm this, that would be really appreciated.

Thanks,

**Comment by dikim33** [comment]
**Comment by bli on 3 Aug 2006 01:09 UTC**
The diff between the 2 rpmlists are:

```
$ diff -ru redhat-3asU2-i386.rpmlist rhel-3-i386.rpmlist
--- redhat-3asU2-i386.rpmlist   2006-07-18 12:38:21.000000000 -0700
+++ rhel-3-i386.rpmlist 2006-07-18 12:38:21.000000000 -0700
@@ -7,6 +7,7 @@
 bash
 bc
 beecrypt
+bind-libs
 bind-utils
 binutils
 bzip2
@@ -71,6 +72,7 @@
 kernel
 kudzu
 krb5-libs
+laus-libs
 laus
 less
 libaio
@@ -131,6 +133,7 @@
 readline
 redhat-logos
 rootfiles
+rpm-libs
 rpm
 rsh
 rsh-server
```

I'm guessing the additional libraries were added in RHEL3 Update 3 and are not present in Update 2 therefore there are 2 RPM lists.  It would be good if you can confirm an installation with RHEL3 Update 2.

If the separate rpmlist is still necessary, we should probably rename the rpmlist to rhel-3u2-i386.rpmlist.

**Comment by dikim33** [comment]
**Comment by dikim on 3 Aug 2006 17:26 UTC**
I don't have RHEL3-u2 machine setup now. If anyone with RHEL3u2 can test this, that would be really great.

**Comment by dikim33** [comment]
**Comment by bli on 4 Aug 2006 02:02 UTC**
Since DongInn does not have the time to test on RHEL3U2, let's just rename the file from 'redhat-3asU2-i386.rpmlist' to 'rhel-3u2-i386.rpmlist' and be done with it.

**Comment by dikim33** [comment]
**Comment by bli on 4 Aug 2006 05:07 UTC**
(In [5282]) This commit is a merge from trunk -> branch-5-0:
   svn merge -r5280:5281 https://svn.oscar.openclustergroup.org/svn/oscar/trunk .

Closes #179.

Original commit message:
Fix for #179 - the rpmlists were renamed to prevent confusion.
